include/oneapi/dpl/internal/common_config.h

```diff
@@ -16,7 +16,13 @@
 #ifndef _ONEDPL_COMMON_CONFIG_H
 #define _ONEDPL_COMMON_CONFIG_H
 
-#if __cplusplus < 201703L
+#ifndef _MSVC_LANG
+#    define _ONEDPL_cplusplus __cplusplus
```

Comment:
```suggestion
#    define _ONEDPL___cplusplus __cplusplus
```

I propose to add two more `_` signs. One is delimeter and other two are a part of `__cplusplus` name

Reply:
If there are no objections from @akukanov @MikeDvorskiy @dmitriy-sobolev side,
I am ready to rename a little bit later in this PR.
Otherwise our discussion on the rest of the comments may be not valid.

Reply:
fixed